lib/attractor/calculators/js_calculator.rb

```diff
@@ -2,8 +2,8 @@
 
 module Attractor
   class JsCalculator < BaseCalculator
-    def initialize(file_prefix: '', minimum_churn_count: 3, start_ago: 365 * 5)
-      super(file_prefix: file_prefix, file_extension: '(js|jsx)', minimum_churn_count: minimum_churn_count, start_ago: start_ago)
+    def initialize(file_prefix: '', ignores: '', minimum_churn_count: 3, start_ago: 365 * 5)
+      super(file_prefix: file_prefix, ignores: ignores, file_extension: '(vue)', minimum_churn_count: minimum_churn_count, start_ago: start_ago)
```

Comment:
please leave the default file extension :-)

Reply:
Ooops... Sorry ;)

I'll re-push as soon as I get home.